<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QWidget" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="black">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </widget>
  <widget class="QLabel" name="white">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </widget>
  <widget class="QLabel" name="blackName">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Player Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="blackTime">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>250</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="whiteTime">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>490</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="whiteName">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>410</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Player Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="blackID">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Player ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="whiteID">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>450</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Player ID:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retractBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>600</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Retract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="concedeBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>600</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Concede</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replayBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>600</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Replay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>600</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
